app/views/PlatformInitiativesListPage.scala.html

```diff
@@ -0,0 +1,137 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import play.api.mvc.Call
+@import uk.gov.hmrc.cataloguefrontend.DateHelper._
+@import uk.gov.hmrc.cataloguefrontend.connector.PlatformInitiative
+@import uk.gov.hmrc.cataloguefrontend.connector.PlatformInitiativesConnector
+@import uk.gov.hmrc.cataloguefrontend.service.PlatformInitiativesService
+@import uk.gov.hmrc.cataloguefrontend.ViewMessages
+@import helper._
+@import scala.concurrent.Future
+@import java.net.URL
+@import scala.util.Try
+
+@this(viewMessages: ViewMessages, platformInitiativesService: PlatformInitiativesService)
+
+@(initiatives          : Seq[PlatformInitiative],
+displayChart           : Boolean,
+displayProgress        : Boolean,
+form                   : Form[_]
+)(implicit messages    : Messages, request: Request[_])
+
+@standard_layout(s"Platform Initiatives", "platforminitiatives") {
+<header>
+    <h1>Platform Initiatives</h1>
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
```

Comment:
CLARIFICATION: Would it be possible to self-host this javascript? Otherwise wouldn't we fall foul of CSP?